ang/civicase/dashboard/directives/dashboard-tab.directive.js

```diff
@@ -46,9 +46,10 @@
         'case_id.contacts'
       ]
     };
+    // The ID is needed for the sort to avoid returning cases in a random order
     var CASES_QUERY_PARAMS_DEFAULTS = {
       'status_id.grouping': 'Opened',
-      options: { sort: 'start_date DESC' },
+      options: { sort: 'start_date DESC, id DESC' },
```

Comment:
If order was wrong in Dashboard, wont it affect Manage Cases too? Also did we find why the order is wrong?

Reply:
Manage cases has something that always appends the `id` parameter to the query. I was searching if there was a note for this, but this was added by core without a note or PR:

https://github.com/compucorp/uk.co.compucorp.civicase/commit/211cdf33e3fdfbb1bc43a236605b1cfa56256cb5#diff-73e074d6c90a338205dbee2d416c987dR277